subsys/caf/events/Kconfig.module_state_event

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config CAF_MODULE_STATE_EVENTS
+	bool "Enable module state events"
+	help
+	  Enable support for module state events.
+
+config CAF_INIT_LOG_MODULE_STATE_EVENTS
+	bool "Log module state events"
+	depends on CAF_MODULE_STATE_EVENTS
+	depends on LOG
+	default y
+
+config CAF_MODULES_FLAGS_COUNT
+	int "Number of supported modules"
+	depends on CAF_MODULE_STATE_EVENTS
+	default 32
+	help
+	  Configure number of supported modules by moules flag variable.
```

Comment:
`module flags`

Reply:
Ok - I get there is error in general, but I am considering if technically more correct would not be "modules flags".